tests/foreman/cli/test_contentview.py

```diff
@@ -763,6 +762,37 @@ def test_positive_remove_version_by_id(self):
         new_cv = ContentView.info({u'id': new_cv['id']})
         self.assertEqual(len(new_cv['versions']), 0)
 
+    @tier1
+    @run_only_on('sat')
+    def test_positive_remove_version_by_name(self):
+        """Delete content view version using 'remove' command by name
+
+        :id: 2c838716-dcd3-4017-bffc-da53727c22a3
+
+        :expectedresults: Content view version deleted successfully
+
+        :BZ: 1416862
+
+        :CaseImportance: Critical
+        """
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': new_cv['id']})
+        new_cv = ContentView.info({u'id': new_cv['id']})
+        self.assertEqual(len(new_cv['versions']), 1)
```

Comment:
I'd add this assert also to the function above (test_positive_remove_version_by_id) in the same context.